scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -730,41 +730,45 @@
                                 (remove #(contains? matched-job-uuids (:job/uuid %)) existing-jobs)))]
     (pc/map-from-keys remove-matched-jobs (keys category->pending-jobs))))
 
+(defn- update-match-with-task-metadata-seq
+  "Updates the match with an entry for the task metadata for all tasks."
+  [{:keys [tasks] :as match} db framework-id executor-config]
+  (let [task-metadata-seq (map #(task/TaskAssignmentResult->task-metadata db framework-id executor-config %)
```

Comment:
There is a lot going on in this expression. Can you switch to use threading?

```clojure
(->> tasks
       (sort-by (comp :db/id :job #(.getRequest ^TaskAssignmentResult %)))
       (map (partial task/TaskAssignmentResult->task-metadata db framework-id executor-config))
```

Reply:
Done, used an anonymous function instead of `partial` as we know exactly how many arguments we will provide.

Reply:
Changed to use `partial`.